src/components/MainMenu/MainMenu.jsx

```diff
@@ -1,25 +1,88 @@
+import React, { Component } from "react";
 import "./MainMenu.css";
-import image1 from './img/IMAGE (7).png';
-import image2 from './img/IMAGE (8).png';
-import image3 from './img/IMAGE (9).png';
-import image4 from './img/IMAGE (10).png';
-import image5 from './img/IMAGE (11).png';
-import image6 from './img/IMAGE (12).png';
 import PhoneTooltip from "./PhoneTooltip";
 
 
 
-const MainMenu = () => {
-        const menuItems = [
-                { id: 1, name: "Burger Dreams", description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry.", price: "$ 9.20 USD", image: image1, counter: 1, button: "Add to card" },
-                { id: 2, name: "Burger Waldo", description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry.", price: "$ 10.00 USD.00", image: image2, counter: 1, button: "Add to card" },
-                { id: 3, name: "Burger Cali", description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry.", price: "$ 8.00 USD", image: image3, counter: 1, button: "Add to card" },
-                { id: 4, name: "Burger Bacon Buddy", description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry.", price: "$ 9.99 USD", image: image4, counter: 1, button: "Add to card"},
-                { id: 5, name: "Burger Spicy", description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry.", price: "$ 9.20 USD", image: image5, counter: 1, button: "Add to card"},
-                { id: 6, name: "Burger Classic", description: "Lorem Ipsum is simply dummy text of the printing and typesetting industry.", price: "$ 8.00 USD", image: image6, counter: 1, button: "Add to card"},
-              ];
+class MainMenu extends Component {
+  constructor(props){
+    super(props);
+    this.state = {
+      menuItems: [],
+      isLoading: true,
+      error: null,
+      visibleItemsCount: 6,
+      cartCount: 0,
+    };
+  }
+  componentDidMount() {
+    fetch("https://65de35f3dccfcd562f5691bb.mockapi.io/api/v1/meals")
+      .then(response => {
+        if (!response.ok) {
+          throw new Error("Ошибка при получении данных с API");
+        }
+        return response.json();
+      })
+      .then(data => {
+        const updatedData = data.map(item => ({ ...item, count: 0 }));
+        this.setState({ menuItems: updatedData, isLoading: false });
+      })
+      .catch(error => {
+        this.setState({ error, isLoading: false });
+      });
+
+  fetch("https://65de35f3dccfcd562f5691bb.mockapi.io/api/v1/orders")
+    .then(res => res.json())
+    .then(data => {
+      console.log("Загрузка заказов:", data);
+    })
+    .catch(err => {
+      console.error("Ошибка при загрузке заказов:", err);
+    });
+  }
+
+  handleSeeMore = () => {
+    this.setState(prevState => ({
+      visibleItemsCount: prevState.visibleItemsCount + 6,
```

Comment:
6 could be moved to constants for it to mean something instead of just being a number